### PR TITLE
complete 7-post-api-reviews-review_id-comments

### PR DESCRIPTION
**Pull request by vincenzochiovaro**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -156,4 +156,26 @@ describe("app ", () => {
...
     });
   });
+  describe("POST /api/reviews/:review_id/comments ", () => {
+    test("respond with 201 when a valid comment is posted", () => {
+      return request(app)
+        .post("/api/reviews/2/comments")
+        .send({ username: "philippaclaire9", body: "test comment" })
+        .expect(201);
+    });
+    test("return status 201, responds with an array of comments for the given review_id ", () => {
```
Comment (itskatherine) [inline __tests__/app.test.js]: This isn't responding with an array of comments, rather the comment that was posted, ✉️  so I would edit this test's description to  reflect that 🙂

File: __tests__/app.test.js [hunk 2]
```diff
@@ -156,4 +156,26 @@ describe("app ", () => {
...
+        .then((comment) => {
+          expect(comment.body).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment.body).toHaveProperty("body", expect.any(String));
+          expect(comment.body).toHaveProperty("review_id", expect.any(Number));
+          expect(comment.body).toHaveProperty("author", expect.any(String));
+          expect(comment.body).toHaveProperty("votes", expect.any(Number));
+          expect(comment.body).toHaveProperty("created_at", expect.any(String));
+        });
+    });
+  });
```
Comment (itskatherine) [inline __tests__/app.test.js:192]: Nice flexible test! Need couple more for error handling (404 for review not found and 400 for invalid review id) 🔧

File: controller.js [hunk 3]
```diff
@@ -34,9 +35,19 @@ const getCommentByReviewId = (request, response) => {
   });
 };
 
+const postComment = (request, response) => {
+  const { review_id } = request.params;
+  const { username, body } = request.body;
```
Comment (itskatherine) [inline controller.js:40]: Nice destructuring makes this very readable!

**Review by itskatherine — APPROVED** [review]
Nice work Vincenzo, check out my comments, there are a couple bits of error handling to add - once these are done feel free to merge 🙂

**Comment by itskatherine** [comment]
Sorry I forgot to add we'll also need a test to make sure we get a 400 if an incomplete comment body is posted like, or one with the wrong keys! Like {usernoom: "kasjdkjds" }

File: __tests__/app.test.js (hunk 2 above)
Comment (vincenzochiovaro) [reply __tests__/app.test.js:192]: Done! thank you for your feedback Katherine

**Pull request merged**